tests/e2e/TestConstants.ts

```diff
@@ -264,5 +269,11 @@ export const TestConstants = {
      /**
       * URL of workspace devfile test repository.
       */
-      TS_TEST_WORKSPACE_DEVFILE_REPO: process.env.TS_TEST_WORKSPACE_DEVFILE_REPO || ''
+      TS_TEST_WORKSPACE_DEVFILE_REPO: process.env.TS_TEST_WORKSPACE_DEVFILE_REPO || '',
+
+    /**
```

Comment:
Cosmetic notice: left code indentation is not equal to those in another code.

Reply:
fixed